src/mtype.d

```diff
@@ -3492,7 +3492,7 @@ public:
             case Tcomplex80:
             case Timaginary80:
             case Tfloat80:
-                fvalue = Port.ldbl_max;
+                fvalue = real.max;
```

Comment:
You introduced a bug, because `Port` is only unconditional for using dmd as the host compiler.